source/includes.js

```diff
@@ -0,0 +1,27 @@
+import _includes from './internal/_includes';
+import _curry2 from './internal/_curry2';
+
+/**
+ * Returns `true` if the specified value is equal, in [`R.equals`](#equals)
+ * terms, to at least one element of the given list; `false` otherwise.
+ * Works also with strings.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.1.0
```

Comment:
`contains` has been around since `0.1.0`, while `includes` has only been around since this PR went into a release.

And at least 3 times now I've countered "includes is not a function" on a project at work, double-checked the Ramda documentation,  and concluded "nope, this is correct and has been since 0.1.0 so it's definitely not because we're not using latest Ramda version."

I don't need to tell you that fixing a mistake is very difficult when you're confidently ruled out the actual solution.

Reply:
This has been fixed on master: https://github.com/ramda/ramda/blob/master/source/includes.js

It will be included in the next documentation release.